alonzo/impl/src/Cardano/Ledger/Alonzo/TxSeq.hs

```diff
@@ -180,16 +181,18 @@ hashTxSeq ::
   Hash (Crypto era) EraIndependentBlockBody
 hashTxSeq (TxSeq' _ bodies ws md vs) =
   coerce $
-    hashStrict
-      ( hashPart bodies
-          <> hashPart ws
-          <> hashPart md
-          <> hashPart vs
-      )
+    hashStrict $
+      fromShort $
```

Comment:
why go by why of a short bytestring here?

Reply:
I was hoping this question would come up :)

That is because `hashToBytes` copies the underlying `ShortByteString` into a `ByteString`, while `hashToBytesShort` is a simple type cast.

In other words we save 4 memory copies when switching to `ShortByteString` and 2 more when switching to `mconcat`.

Reply:
oh great, thank you for explaining!